src/test/java/de/tum/in/www1/artemis/exercise/fileupload/FileUploadExerciseIntegrationTest.java

```diff
@@ -100,6 +107,12 @@ class FileUploadExerciseIntegrationTest extends AbstractSpringIntegrationIndepen
     @Autowired
     private PageableSearchUtilService pageableSearchUtilService;
 
+    @SpyBean
+    private ExamLiveEventsService examLiveEventsService;
+
+    @SpyBean
+    private GroupNotificationScheduleService groupNotificationScheduleService;
+
```

Comment:
Only use SpyBeans in a base test class (i.e. `AbstractSpringIntegrationIndependentTest`)
Otherwise this will lead to another start of the entire application context. This takes ~30s each, so we try to minimize it as much as possible (currently 4).

Reply:
done